sequencing-server/src/app.ts

```diff
@@ -505,7 +497,7 @@ app.post('/get-seqjson-for-sequence-standalone', async (req, res, next) => {
     const context: Context = res.locals['context'];
     commandTypes = await context.commandTypescriptDataLoader.load({ dictionaryId: commandDictionaryID });
   } catch (e) {
-    res.status(200).json({
+    res.status(500).json({
```

Comment:
There seems to be a mix of returning 200s and 500s in this file when catching. Was this 200 here for a reason before?

Reply:
Good point! I think we can rework the this error mode in general. Right now, we catch all errors from the dataloader, which includes command dictionary not found errors as well as network errors. The command dictionary not found errors fall along the lines of the "business logic driven errors" that should be reflected in fallable error returns (which use 200). The network errors we want to just throw and be auto handled by hasura.

Now a complicating element here is that the command dictionary not found error is NOT the error type we specify in the graphql api [here](https://github.com/NASA-AMMOS/aerie/blob/feature/489--bulk-seqJson/deployment/hasura/metadata/actions.graphql#L181) so ideally we'd have a union error type for the API where we include a CommandDictionaryNotFound error type (or a generic error type) that at a minimum does not contain a code location as the UserCodeError type does as its definitely not relevant to the error of CommandDictionaryNotFound. I don't think Hasura allows user created union types, so we can't currently do that.

Because of this, I think it's best to return as a 500 so that it circumvents the shape represented in our graphql api.

We should probably add failure mode test cases at some point (probably a good intro task for onboarding).